packages/react-component-playground/src/components/component-playground.jsx

```diff
@@ -213,11 +218,19 @@ module.exports = React.createClass({
         key="previewContainer"
         className={this.getPreviewClasses()}
       >
-        {this.loadChild('preview')}
+        {this.renderComponent()}
       </div>
     );
   },
 
+  renderComponent() {
+    return _.reduce(this.props.proxies, (accumulator, proxy) =>
+      React.createElement(proxy.component, _.assign({}, this.props, {
+        children: React.Children.only(accumulator),
+        ref: proxy.ref,
```

Comment:
It's not needed right now (except in tests), but I'm thinking we should have a way to access these proxies in the future.